ledger-core/virtual/integration/vstatereport_test.go

```diff
@@ -92,3 +99,368 @@ func TestVirtual_SendVStateReport_IfPulseChanged(t *testing.T) {
 
 	require.Equal(t, 1, countVStateReport)
 }
+
+func TestVirtual_StateReport_CheckPendingCountersAndPulses(t *testing.T) {
+	t.Log("C FIXME ")
```

Comment:
I don't know all test naming rules, but is it ok? @ElSacramento

Reply:
  C5111